src/components/navigation/section-document-or-browser.tsx

```diff
@@ -196,20 +199,51 @@ export const SectionDocumentOrBrowser: React.FC<IProps> = ({ tabSpec, reset, sel
   };
 
   const showPlayback = user.isTeacher;
+  const [showPlaybackControls, setShowPlaybackControls] = useState(false);
+  const [documentToShow, setDocumentToShow] = useState<DocumentModelType>();
+
+  useEffect(() => {
+    setDocumentToShow(referenceDocument);
+  },[referenceDocument]);
+
+  const handleTogglePlaybackControlComponent = () => {
+    setShowPlaybackControls(!showPlaybackControls);
+    const newState = !showPlaybackControls;
+    setDocumentToShow(newState
+                        ? getDocumentToShow()
+                        : referenceDocument
+                      );
+  };
+
+  const getDocumentToShow = () => {
+    if (referenceDocument) {
+      const origDocManager = referenceDocument.treeManagerAPI as Instance<typeof TreeManager>;
+      const docCopy = DocumentModel.create(getSnapshot(referenceDocument));
+      const historySnapshot = (getSnapshot(origDocManager.document)) as unknown as CDocumentType;
+      const docCopyManager = docCopy.treeManagerAPI as Instance<typeof TreeManager>;
+      docCopyManager.setChangeDocument(historySnapshot);
+      docCopyManager.setCurrentHistoryIndex(origDocManager.document.history.length);
+      return docCopy;
```

Comment:
This is not covered by tests. I'd guess the best approach would be to use cypress to do some coverage of the time travel slider.

This is a note to myself: 
> I'd like to add a new property to the treeManager which disables its recording of events, when it is used on these kinds of documents.